Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelSelectTypePage.xaml.cs

```diff
@@ -210,6 +210,15 @@ private void APIType_SelectionChanged(object sender, SelectionChangedEventArgs e
             }
             xPreviewButton.Visibility = Visibility.Collapsed;
             SourceRviewLable.Visibility = Visibility.Collapsed;
+
+            if (AddAPIModelWizard != null)
+            {
+                AddAPIModelWizard.IsParsingWasDone = false;
```

Comment:
1- please use block {} also when you have only one line in IF 
2- please create function called "CleanLearnedAPIModels()" and put the cleaning code in it and call the function when needed instead duplicating the code

Reply:
Done !